flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -227,6 +227,12 @@
 	public static final String TASK_MANAGER_NETWORK_NUM_BUFFERS_KEY = "taskmanager.network.numberOfBuffers";
 
 	/**
+	 * Boolean flag to enable/disable more detailed metrics about inbound/outbound network queue lengths
+	 */
+	@PublicEvolving
+	public static final String NETWORK_DETAILED_METRICS_KEY = "taskmanager.net.detailed-metrics";
```

Comment:
based on the config key just above this one "taskmanager.network..." would be more appropriate.